views/ServiceSelector.qml

```diff
@@ -5,6 +5,7 @@ ComboBox
 {
     textRole: "label"
     model: ThingiService.drivers
+    currentIndex: ThingiService.getDriverIndex(ThingiService.activeDriver)
```

Comment:
Isn't it possible to derive the index in javascript here? The Python back-end doesn't really have much to do with it.

Reply:
It definitely can and I had it that way at one point but I also had to use it in a couple of places so I put in python to have the code reusable in one place. I should create a js function here and in ThingiHeader?

Reply:
Other option would be to wrap the combobox in a component that has the function in it? That way it's less code duplication.